src/pages/SignUp/SignUp.js

```diff
@@ -1,8 +1,213 @@
-import React from 'react'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import TermsList from './TermsList'
+import { TERMS_BOX } from './termsBox'
+import './SignUp.scss'
 
 const SignUp = () => {
-  return <h1>hello</h1>
+  const navigate = useNavigate()
+
+  const [email, setEmail] = useState('')
+  const [name, setName] = useState('')
+  const [password, setPassword] = useState('')
+  const [passwordConfirm, setPasswordConfirm] = useState('')
+
+  const [isEmail, setIsEmail] = useState(false)
+  const [isName, setIsName] = useState(false)
+  const [isPassword, setIsPassword] = useState(false)
+  const [isPasswordConfirm, setIsPasswordConfirm] = useState(false)
+  const [isMatched, setIsMatched] = useState(true)
+  const [isAllValid, setIsAllValid] = useState(false)
+
+  const [isOpenTerms, setIsOpenTerms] = useState([false, false, false])
+  const [checkItems, setCheckItems] = useState([])
+
+  const signupHandler = () => {
+    fetch('', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json;charset=utf-8' },
+      body: JSON.stringify({
+        email: email,
+        password: password,
+        name: name,
+      }),
+    })
+      .then(response => {
+        return response.json()
+      })
+      .then(response => {
+        if (response.success === true) {
```

Comment:
```suggestion
        if (response.success) {
```
해당 값 자체가 boolean 값이기 때문에 위와 같이 변경할 수 있겠습니다!